docker-compose.yml

```diff
@@ -26,44 +25,51 @@ services:
     ports:
       - "9042:9042"
   couchbase:
-    image: couchbase:community-5.1.1
+    image: couchbase:community-7.6.1
     ports:
-      - "8091-8094:8091-8094"
-      - "11210:11210"
+      - "8091-8096:8091-8096"
+      - "11210-11211:11210-11211"
```

Comment:
didn't notice this on the previous PR, Is it verified?

Reply:
* this change is from #624 
* the couchbase tests pass when I copy back this change but they fail if I leave the values as is
* the tests are not run on the 1.0.x branch unless we have a PR and unfortunately the GitHub CI changes to remove support for Docker v1 require us to update some of our docker-compose configs
* It's possible that the port changes are not all needed but I thought it was better to not customise the changes and just to try to keep the docker config that works in the main branch and use it in the 1.0.x branch too